docs/documents/querying/linq/async-enumerable.md

```diff
@@ -40,5 +40,10 @@ public async Task query_to_async_enumerable()
     }
 }
 ```
+
+::: warning
+Be aware not to return the IAsyncEnumerable out of the scope in which the session that produces it is used. This would prevent the data base connection from being reused afterwards and thus leading to a connection bleed.
+:::
+
```

Comment:
```suggestion

::: warning
Be aware not to return the IAsyncEnumerable out of the scope in which the session that produces it is used. This would prevent the database connection from being reused afterwards and thus lead to a connection bleed.
:::

```